helpers/controllers/userInfoController.ts

```diff
@@ -22,6 +26,23 @@ export const userInfo = async (
   if (!user) {
     throw new Error('Invalid user object')
   }
+
+  let discordUsername = '',
+    discordAvatarUrl = ''
+  if (user.discordRefreshToken) {
+    try {
+      const discordUserInfo = await getUserInfoFromRefreshToken(
+        user.id,
+        user.discordRefreshToken
+      )
+
+      discordUsername = _.get(discordUserInfo, 'username', '')
+      discordAvatarUrl = _.get(discordUserInfo, 'avatarUrl', '')
```

Comment:
can we get rid of lodash here and just use the newer js syntax  ` discordUsername = discordUserInfo?. username ??  ''`